spec/bookshelf-spec.ts

```diff
@@ -616,6 +616,133 @@ describe('Bookshelf links', () => {
     expect(_.matches(expected)(result)).toBe(true);
   });
 
+  it('should not add pagination links if no pagination data is passed', () => {
+    let limit: number = 10;
+    let offset: number = 40;
+    let total: number = 100;
+
+    let elements: Model[] = _.range(10).map((num: number) => {
+      return bookshelf.Model.forge<any>({id: num, attr: 'value' + num});
+    });
+
+    let collection: Collection = bookshelf.Collection.forge<any>(elements);
+
+    let result: any = mapper.map(collection, 'models');
+
+    expect(result.links).not.toContain(['first', 'prev', 'next', 'last']);
```

Comment:
Same problem here with passing an array to `toContain`, pass each argument in a new call.